src/ui/Combobox/ComboboxInput.ts

```diff
@@ -0,0 +1,98 @@
+import { TextInput, ITextInputOptions } from '../FormWidgets/TextInput';
+import { $$ } from '../../utils/Dom';
+import { Combobox } from './Combobox';
+import { KEYBOARD } from '../../utils/KeyboardUtils';
+import { Utils } from '../../utils/Utils';
+
+export interface IComboboxAccessibilityAttributes {
+  activeDescendant: string;
+  expanded: boolean;
+}
+
+export class ComboboxInput {
```

Comment:
Would that be usable for other stuff than just facet search ?

If yes, then the name makes sense, otherwise, there should be something in it's name that mention that it can be only used for facet(s) search.

Reply:
Yes for anything really though the focus is more for an async list of "results" through the API